src/components/bottom-control-buttons/components/VolumeButton.tsx

```diff
@@ -33,9 +33,9 @@ export const VolumeButton: FC<VolumeButtonProps> = ({
 
 	const onToggleMute = () => {
 		if (isMuted) {
-			unmute();
+			return unmute();
```

Comment:
This would be easier to read as:
```ts
return isMuted ? unmute() : mute();
```
because `isMuted` isn't really a special case that requires early exit - but just one of two equal options.